src/helpers.js

```diff
@@ -157,3 +176,39 @@ export function getCenter(a, b) {
     y: (b.y + a.y) / 2,
   }
 }
+
+function rotate(x, y, xm, ym, a) {
```

Comment:
Could you use more verbose variable names so that it's easier to understand what each of them do? 🙂